File: src/main/java/com/kmatrokhin/uvbot/UvIndexBotApplication.java
package com.kmatrokhin.uvbot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class UvIndexBotApplication {

	public static void main(String[] args) {
		SpringApplication.run(UvIndexBotApplication.class, args);
	}

}

File: src/main/java/com/kmatrokhin/uvbot/UvIndexBotApplication.kt
package com.kmatrokhin.uvbot

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cloud.openfeign.EnableFeignClients
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
@EnableFeignClients
class UvIndexBotApplication

fun main(args: Array<String>) {
    SpringApplication.run(UvIndexBotApplication::class.java, *args)
}
